src/sections/Community/index.js

```diff
@@ -100,25 +100,27 @@ const CommunityPage = () => {
           </Row>
         </Container>
         <Row className="newcomers-section">
-          <Col sm={6} lg={6} className="map">
-            <NewcomersMap />
-          </Col>
-          <Col sm={6} lg={6}>
+          <Container>
```

Comment:
Instead of using the `Container` inside it, wrap the whole thing (which is the `Row`) in the above `Container` ending at line: 101.

Reply:
I tried doing that, but the grey background followed along the margins as well, a bit off the design.